tests/unittest/algorithms.cpp

```diff
@@ -0,0 +1,44 @@
+#include "catch2/catch_test_macros.hpp"
+#include "Algorithms.h"
+
+TEST_CASE("Mass", "[algorithms]") {
+  ROOT::VecOps::RVec<edm4hep::ReconstructedParticleData> pVec;
+  edm4hep::ReconstructedParticleData p;
+  p.momentum.x = 1.;
+  p.momentum.y = -1.;
+  p.momentum.z = 1.;
+  p.energy = 2.;
+  pVec.push_back(p);
+  REQUIRE(Algorithms::getMass(pVec) == 1.);
```

Comment:
```suggestion
  REQUIRE(Algorithms::getMass(pVec) == Approx(1.));
```
Careful with floating point comparisons, see https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#floating-point-comparisons